### DISCUSSION TITLE
Can I open to browser currently open?

### DISCUSSION THREAD

**Discussion by adrianoresende** [johnste/finicky#316] [General]
Example:
The default is Safari, but if I close Safari and open Chrome, I want the Mail open to the browser currently, it means that it must open Chrome and don't Safari.

**Comment by davidalim** [comment]
I had this working before v4 using 
`defaultBrowser: ["Firefox", "Google Chrome"], // Use browsers that are open, in this order`

but in V4 I get an error that it can't be an array.

**Reply by johnste** [reply]
You can use `finicky.isAppRunning(...)` to check for currently open applications. 

See [this comment](https://github.com/johnste/finicky/issues/145#issuecomment-2923027936) for an example that replicates the earlier feature
